nexus/bin/qdens-radial

```diff
@@ -0,0 +1,513 @@
+            self.log('Resampling to obtain error bar (NOTE: This can be slow)...')
+            if opt.vmc_file is not None and opt.vmc_err_file is not None:
+                # Data without errors
+                vmc_xsf_data = XsfFile(filepath=opt.vmc_file)
+                dmc_xsf_data = XsfFile(filepath=self.file_in)
+                # Data with errors
+                vmc_plus_err_xsf_data = XsfFile(filepath=opt.vmc_err_file)
+                dmc_plus_err_xsf_data = XsfFile(filepath=opt.dmc_err_file)
+                # Errors only
+                vmc_err_xsf_data = vmc_plus_err_xsf_data.copy() 
+                vmc_err_xsf_data = vmc_plus_err_xsf_data.data[3]['density']['density'].values \
+                                   - vmc_xsf_data.data[3]['density']['density'].values
+                dmc_err_xsf_data = dmc_plus_err_xsf_data.copy() 
+                dmc_err_xsf_data = dmc_plus_err_xsf_data.data[3]['density']['density'].values \
+                                   - dmc_xsf_data.data[3]['density']['density'].values
+
+                # Get standard deviation corresponding to the extrapolation formula: 
+                #   
+                # std_dev = sqrt(4*sigma_dmc**2+sigma_vmc**2)
+                extrap_err_xsf_data = dmc_err_xsf_data.copy()
+                extrap_err_xsf_data.data[3]['density']['density'].values = np.sqrt(4.0*dmc_err_xsf_data.data[3]['density']['density'].values**2.0+vmc_err_xsf_data.data[3]['density']['density'].values**2.0)
+                err_data = extrap_err_xsf_data.copy()
+            elif opt.vmc_file is None:
+                # Data without errors
+                dmc_xsf_data = XsfFile(filepath=self.file_in)
+                # Data with errors
+                dmc_plus_err_xsf_data = XsfFile(filepath=opt.dmc_err_file)
+                # Errors only
+                dmc_err_xsf_data = dmc_plus_err_xsf_data.copy() 
+                dmc_err_xsf_data.data[3]['density']['density'].values = dmc_plus_err_xsf_data.data[3]['density']['density'].values \
+                                   - dmc_xsf_data.data[3]['density']['density'].values
+                err_data = dmc_err_xsf_data.copy()
+
+            else:
+                self.error('VMC and DMC densities provided but only one +err file was given.\nPlease either provide no +err file (which avoids resampling) or\nprovide a +err file for both VMC and DMC with --vmc_err_file and --dmc_err_file arguments.')
+            #end if
+            show_error=True
+
+            # We now have the means and the errors (either extrapolated or not). So let's now resample for the errorbars
+            
+            sum_dict = {}
+            sum_sq_dict = {}
+            std_dict = {}
+
+            nsamples = opt.nsamples
+            self.log('Will compute {} samples...'.format(nsamples))
+            for i in range(nsamples):
+                self.log('sample: {}'.format(i))
+                resample_xsf_data = file.copy()
```

Comment:
I use the file object s.t. it can be fed into the calculation of the moments or occupations in order to sample those quantities and get their error bars.